test/snippet/search/Algorithm/search.cpp

```diff
@@ -0,0 +1,16 @@
+#include <seqan3/alphabet/nucleotide/dna4.hpp>
+#include <seqan3/core/debug_stream.hpp>
+#include <seqan3/search/algorithm/search.hpp>
+#include <seqan3/search/fm_index/all.hpp>
+int main()
```

Comment:
❗️  Please add the include for std::vector.
💅  I would insert an empty line between the includes and main function.